blogs.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Page Title</title>
```

Comment:
Make sure to fill out these page titles.

Also indent everything inside of the head